roles/zabbix_agent/tasks/Linux.yml

```diff
@@ -75,11 +75,13 @@
 
 - name: "Fail invalid specified agent_listenip"
   ansible.builtin.fail:
-    msg: "The agent_listenip does not exist"
+    msg: "{{ item }} The agent_listenip does not exist"
   when:
-    - zabbix_agent_listenip != '0.0.0.0'
-    - zabbix_agent_listenip != '127.0.0.1'
-    - (zabbix_agent_listenip not in ansible_all_ipv4_addresses)
+    - item != '0.0.0.0'
+    - item != '127.0.0.1'
+    - item not in ansible_all_ipv4_addresses
+  with_items:
+    "{{ zabbix_agent_listenip | split(',') }}"
```

Comment:
We need to separate with `;` for HA.

https://www.zabbix.com/documentation/6.0/en/manual/concepts/server/ha

Reply:
I am sorry, this is listening IP on the agent side? I don't think it can have a semicolon https://www.zabbix.com/documentation/7.0/en/manual/appendix/config/zabbix_agent2#listenip

Reply:
Oh, sorry.
I understand it.